governance/polls/SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024.md

```diff
@@ -0,0 +1,62 @@
+---
+title: SparkLend Ethereum - Activate Lido Liquid Staking (LST) Interest Rate Model (IRM) - July 29, 2024
+summary: Signal your support or opposition to activating the Lido Liquid Staking (LST) yield oracle and the linked Interest Rate Model (IRM) for the ETH market on SparkLend.
+discussion_link: https://forum.makerdao.com/t/jul-27-2024-proposed-changes-to-spark-for-upcoming-spell/24755
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-07-29T16:00:00
+end_date: 2024-08-01T16:00:00
+---
+# Poll: SparkLend Ethereum - Activate Lido LST Interest Rate Model (IRM) - July 29, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, July 29 at 16:00 UTC.
+
+**This is a binary vote.**
+
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to activating the Lido LST yield oracle and the linked Interest Rate Model for the ETH market on SparkLend with the following parameters:
```

Comment:
Abbreviations should be reviewed here. The title introduces LST and IRM as abbreviations. I would suggest they both still be written "Liquid Staking (LST)" and "Interest Rate Model (IRM)". Regardless, the sentence should make sure the same format is used for LST and IRM.